include/odp/api/spec/ipsec.h

```diff
@@ -70,6 +70,48 @@ typedef enum odp_ipsec_op_mode_t {
 
 } odp_ipsec_op_mode_t;
 
+/**
+ * IPSEC DEBUG SA parameter type
+ */
+typedef enum odp_ipsec_debug_sa_param_type_t {
+	/** Sequence number */
+	ODP_IPSEC_DEBUG_SA_FIELD_SEQ_NUM = 0,
+
+	/* Antireplay window top */
+	ODP_IPSEC_DEBUG_SA_FIELD_ANTIREPLAY_WINDOW_TOP
+
+} odp_ipsec_debug_sa_param_type_t;
```

Comment:
The names seem slightly awkward. I think the _FIELD part could be dropped.

> /** Sequence number */
I think this should be "Next sequence number".

> /* Antireplay window top */
This may also be a bit ambiguous. Maybe the comment could be "Highest authenticated sequence number"?

Reply:
Ack.